### ISSUE TITLE
Issues Installing Plugin 

### ISSUE THREAD

**Issue by austenc** [rocketeers/rocketeer-slack#26]
Hello,

I'm using Laravel 4 and the latest version of rocketeer and did a fresh `ignite` after I installed it. I configured everything and was able to deploy. After that I have been attempting to get the slack plugin to work without much success.

There seems to be some discrepancy between the latest plugin code and rocketeer's documentation regarding plugins? I'm going to list a few issues I've been having in hopes of getting some feedback on any/all of them from you @Anahkiasen . Note that I am happy to contribute and submit some pull requests as needed to bring more clarity to this section of rocketeer, I'm just looking for a little direction! 
- When looking at [rocketeer's docs on plugins](http://rocketeer.autopergamene.eu/#/docs/docs/II-Concepts/Plugins), it says to install plugins via `rocketeer plugin:install <package>`. I recall this is how it worked last time I setup rocketeer (~8 months ago). When I run this command from my project folder, it doesn't install the plugin to the correct place. My folder is at `/home/vagrant/Code/myproject` and the plugin installs to `/home/vagrant/.rocketeer/plugins/xyz` ... so something isn't working as I would expect here.
- After trying the above "old method", I went to the [README](https://github.com/rocketeers/rocketeer-slack) for the rocketeer-slack plugin and attempted to follow the directions there. When I ran the `artisan config:publish rocketeer/rocketeer-slack` command, I received a `[InvalidArgumentException] Configuration not found.` error. I _was_ able to publish the configs via `artisan config:publish anahkiasen/rocketeer-slack`. Once I published the config and entered my slack URL, I was greeted with an `string expected, array given` error. Digging in further, within the `RocketeerSlack` class, it wasn't able to find the configured url, `$app['config']->get('rocketeer-slack::url')` returns `array()`. 

Overall, for now I have just scrapped the idea of the slack plugin, but I'd love to get it working again and/or contribute to rocketeer if possible!

**TL;DR - tried installing slack plugin the 'old' way and it didn't work, tried following readme from the plugin itself and encountered errors as well.**
